scripts/vsc-venv-wrapper/README.md

```diff
@@ -0,0 +1,95 @@
+# Python Virtual Environment Wrapper Script
+
+`vsc-venv` is a script that encapsulates the creation and management of Python virtual environments. 
+This avoids multiple issues with the default `venv` command on the HPC. 
+
+
+One issue is that a virtual environment created for one cluster might not work on another. 
+Additionally, when activating a virtual environment, 
+the same centrally installed modules used during its creation must also be loaded.
+the `vsc-venv` command manages multiple virtual environments for different clusters in a transparent way, 
+while guaranteeing the same module environment.
+
+## Usage
+
+A virtual environment can be activated by running the following command:
+
+```bash
+source ./vsc-venv.sh --activate --requirements <requirements> [--modules module_file]
```

Comment:
Change all occurrences of `./vsc-venv.sh` to `module load vsc-env`, and just using `source vsc-venv` (no `./`).

We'll install the `vsc-env` module with versioning.

Reply:
Done in [use 'module load vsc-env'](https://github.com/hpcugent/vsc_user_docs/pull/670/commits/66e03c36b62b6c48335af3322ee07ad89c848855), except when deactivate is called, as the module is already loaded at that time

Reply:
forgot to [remove .sh](https://github.com/hpcugent/vsc_user_docs/pull/670/commits/241c3dc0698912404434e90a4294cb45d3ac83b7)

Reply:
is it right that the module is not called `vsc-venv` but `vsc-env`?